WNPRC_EHR/resources/queries/study/remainingWater/.qview.xml

```diff
@@ -0,0 +1,17 @@
+<customView xmlns="http://labkey.org/data/xml/queryCustomView">
+   <columns>
+       <column name="Id" />
+       <column name="date" />
+       <column name="volumeGivenByLab" />
```

Comment:
@dnicolalde do these columns match up to the query? In the remainingWater.sql file I noticed here it is called 'volumeGivenByLab' but in the query it maybe is called 'volumeGivenInLab' ?

Reply:
this an older code deleted from the branch.